python/hail/expr/functions.py

```diff
@@ -1796,6 +1796,7 @@ def _num_allele_type(ref, alt) -> Int32Expression:
     return hl.bind(lambda r, a:
                    hl.cond(_is_valid_allele(r) & _is_valid_allele(a),
                            hl.case()
+                           .when((ref[0] == '<') | (alt[0] == '<'), _allele_ints['Symbolic'])
```

Comment:
Should we allow the ref allele to be symbolic? (Also seems strange to call this case AltAlleleType, but may be simplest)

Reply:
this is called `allele_type` in python, not `alt_allele_type`

Reply:
and I have no idea whether we should allow the ref to be symbolic.